### ISSUE TITLE
Unable to make cover image text appear bold in the editor.

### ISSUE THREAD

**Issue by kjellr** [WordPress/gutenberg#8202]
**Describe the bug**

It's currently impossible make cover image text appear bold inside of the editor. If you select the bold toggle, there's a CSS rule that overrides it:

https://github.com/WordPress/gutenberg/blob/5f5ae541d30d4670da0e3118e5e71517fe0692d0/core-blocks/cover-image/editor.scss#L16

**To Reproduce**

1. Insert a cover image block.
2. Write some text inside of it. 
3. Use the inline option to make the text bold.

**Expected behavior**

I expected the text to appear bold, but it was not. The text _does_ appear bold on the front end. 

**Screenshots**

![cover-image-bold](https://user-images.githubusercontent.com/1202812/43211737-3cd86ace-9000-11e8-98de-75bb50dba474.gif)

**Additional context**
Gutenberg v3.3.0

**Comment by designsimply** [comment]
Duplicate of #4405 (but it's possible I closed that one prematurely! so please have a look at the related discussion). 🙂

**Comment by kjellr** [comment]
@designsimply They're really similar bugs, but that one is a little out of date. In that bug:

- The text in the header image block is bold by default. But when you select the "bold" option, it changes to a lighter weight.

But for this bug:

- The text in the header image block is _not_ bold by default (that must've changed at some point since that last bug was reported). When you select the "bold" option, nothing happens.

**Comment by designsimply** [comment]
> When you select the "bold" option, nothing happens

When you highlight the text and select bold, `<strong>` tags are added. You're right that it appears nothing happens visually though! My interpretation of this comment is that cover image block text is styled as bold by default and that's why it doesn't appear to change but themes may want to surface the bold behavior and so the bold button adds html for that.

**Comment by kjellr** [comment]
> When you highlight the text and select bold, `<strong>` tags are added. You're right that it appears nothing happens visually though! My interpretation of this comment is that cover image block text is styled as bold by default and that's why it doesn't appear to change but themes may want to surface the bold behavior and so the bold button adds html for that.

Yep, that's all right, except that the Cover Image text is no longer bold by default — its weight is set to `300`. Since `<strong>` tags are added, the fact that nothing happens visually is really the core of this issue. 🙂

(All that said, I think this issue will be resolved if we convert the Cover image block to nested blocks, as described in https://github.com/WordPress/gutenberg/pull/5452.)
